toolchain/check/pattern_match.h

```diff
@@ -10,15 +10,14 @@
 
 namespace Carbon::Check {
 
-// The pattern-match counterparts of the patterns passed to CalleePatternMatch.
+// The outputs of CalleePatternMatch.
+// TODO: rename or remove this struct.
```

Comment:
```suggestion
// TODO: Rename or remove this struct.
```

Since we've been talking about TODO comments a lot, I'll note all the style examples at https://google.github.io/styleguide/cppguide.html#TODO_Comments use the same sentence capitalization; it probably makes sense to be consistent.